### ISSUE TITLE
split dataset to training, validation, and test sets

### ISSUE THREAD

**Issue by williamstark01** [yangtcai/Ensembl-Repeat-Identification#44]
Simply replicate how the validation set is currently created, or take a look at another method here:
https://github.com/Ensembl/gene_pcp/blob/main/utils.py#L260
